pkg/datastore/db.go

```diff
@@ -90,23 +90,23 @@ type SQLClient struct {
 //nolint:gochecknoglobals // sqlStats has to be a global variable for prometheus
 var (
 	sqlStats = prometheus.NewHistogramVec(prometheus.HistogramOpts{
-		Name:    "zs_sql_stats",
+		Name:    "gofr_sql_stats",
```

Comment:
instead of updating like this, we can use `FrameworkMetricsPrefix` constant, just like PubSub metrics.

Reply:
Absolutely @srijan-27 was just testing if the tests and code are working, will update it in final commit.